### PR TITLE
consistency pass #2

### PR DESCRIPTION
**Pull request by mmacy**
Snapping to `ACCOUNT_URI` and `AUTH_URI` (instead of `*_HOST`) to better align with [Cosmos DB terminology](https://docs.microsoft.com/en-us/rest/api/cosmos-db/cosmosdb-resource-uri-syntax-for-rest).

Also a few more updates to comments in the example *.py files.

### CODE REVIEW

File: examples/documentmanagementsample.py [hunk 1]
```diff
@@ -111,11 +111,11 @@ def GetSalesOrderV2(document_id):
...
                     {'order_qty' : 3,
-                     'product_code' : 'A-123',      # notice how in item details we no longer reference a ProductId
-                     'product_name' : 'Product 1',  # instead we have decided to denormalise our schema and include 
-                     'currency_symbol' : '$',       # the Product details relevant to the Order on to the Order directly
-                     'currecny_code' : 'USD',       # this is a typical refactor that happens in the course of an application
-                     'unit_price' : 17.1,           # that would have previously required schema changes and data migrations etc.
+                     'product_code' : 'A-123',      # Notice how in item details we no longer reference a ProductId.
+                     'product_name' : 'Product 1',  # Instead, we've decided to de-normalize our schema and include
+                     'currency_symbol' : '$',       # the Product details relevant to the Order on the Order directly.
+                     'currecny_code' : 'USD',       # This is a typical refactor that happens in the course of an application
```
Comment (johanste) [inline examples/documentmanagementsample.py]: curre**cn**y => curre**nc**y?

Comment (mmacy) [reply examples/documentmanagementsample.py]: Good catch, fixed.

**Comment by mmacy** [comment]
@johanste Will resolve, stand by.

**Comment by mmacy** [comment]
@johanste OK, this should be ready for your review.

**Comment by johanste** [comment]
LGTM!

**Pull request merged**